File: JAVA/Program/Komponen.java
package Program;
public class Komponen {
    private String merk;
    private String nama;

    public Komponen() {
        // Konstruktor default
    }

    public Komponen(String merk, String nama) {
        this.merk = merk;
        this.nama = nama;
    }

    public void setMerk(String merk) {
        this.merk = merk;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public String getMerk() {
        return merk;
    }

    public String getNama() {
        return nama;
    }
}

File: PYTHON/Program/Komponen.py
class Komponen:
    def __init__(self, merk="", nama=""):
        self._merk = merk
        self._nama = nama

    def setMerk(self, merk):
        self._merk = merk

    def setNama(self, nama):
        self._nama = nama

    def getMerk(self):
        return self._merk

    def getNama(self):
        return self._nama
